05-interactive-dst.md

```diff
@@ -181,3 +181,23 @@ print_decay.printTree(cands[0])
 With our candidates in hand, it would be nice to be able to retrieve and 
 compute the variables we need for an analysis. On to [LoKi 
 functors](06-loki-functors.html)!
+
+> ## Fast DST browsing {.callout}
+>
+> While here we have discussed for pedagogical reasons all the configuration
+> options needed in order to browse a `DST` file, in your daily life as a
+> physicist is often useful to use the `bender` application that belongs to
+> the `Bender` project.
+>
+> For example, to explore the `DST` we could have simply done:
+>
+> ```
+> lb-run Bender v27r0p1 bender 00035742_00000001_1.allstreams.dst
+> ```
```

Comment:
True, I'll remove it :-)

On Thu, Dec 3, 2015 at 1:17 PM, Alex Pearce <notifications@github.com>
wrote:

> In 05-interactive-dst.md
> <https://github.com/lhcb/first-analysis-steps/pull/160#discussion_r46544394>
> :
>
> > @@ -181,3 +181,23 @@ print_decay.printTree(cands[0])
> >  With our candidates in hand, it would be nice to be able to retrieve and
> >  compute the variables we need for an analysis. On to [LoKi
> >  functors](06-loki-functors.html)!
> > +
> > +> ## Fast DST browsing {.callout}
> > +>
> > +> While here we have discussed for pedagogical reasons all the configuration
> > +> options needed in order to browse a `DST` file, in your daily life as a
> > +> physicist is often useful to use the `bender` application that belongs to
> > +> the `Bender` project.
> > +>
> > +> For example, to explore the `DST` we could have simply done:
> > +>
> > +> ```
> > +> lb-run Bender v27r0p1 bender 00035742_00000001_1.allstreams.dst
> > +> ```
>
> Is there a need to specify a Bender version here? This step probably
> doesn't need to be ‘reproducible’ like the DV stuff.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lhcb/first-analysis-steps/pull/160/files#r46544394>.
>



-- 
Dr. Albert Puig Navarro
Laboratoire de Physique des Hautes Energies
Ecole Polytechnique Fédérale de Lausanne (EPFL)
BSP 614.4 (Cubotron UNIL) CH-1015 Lausanne
EPFL Phone: 021 6939808
CERN Phone: 72518